jdaviz/configs/cubeviz/plugins/viewers.py

```diff
@@ -16,7 +17,7 @@
 
 
 @viewer_registry("cubeviz-image-viewer", label="Image 2D (Cubeviz)")
-class CubevizImageView(JdavizViewerMixin, BqplotImageView):
+class CubevizImageView(JdavizViewerMixin, BqplotImageView, AstrowidgetsImageViewerMixin):
```

Comment:
The viewer itself is an image viewer, with the image that is displayed being controlled by the slice.  If you want to add those tests now and see what can't work, great, otherwise maybe we defer the cubeviz support for now, and image viewers in cubeviz just don't get the exposed astrowidgets zoom functionality until we get around to that?

Reply:
I'll be happy to once I free myself from the rotation review stuff, so not this sprint. Do you want me to make a ticket? But I agree that until we can confirm the Astrowidgets API works on Cubeviz, we should not expose it.

Reply:
I'll remove it from cubeviz for the sake of this PR and we can revisit for cubeviz later.

Reply:
OK. I created https://jira.stsci.edu/browse/JDAT-3981